napi_js_data.md

```diff
@@ -0,0 +1,873 @@
+# N-API - JavaScript Data Types and Values 
+
+N-API exposes a set of APIs to create all types of JavaScript values. 
+Some of these types are documented under 
+[Section 6](https://tc39.github.io/ecma262/#sec-ecmascript-data-types-and-values) 
+of the [ECMAScript Language Specification](https://tc39.github.io/ecma262/).
+
+Fundamentally, these APIs are used to do one of the following:
+1. Create a new JavaScript object
+2. Convert from a primitive C++ type to an N-API value
+3. Convert from N-API value to a primitive C++ type
```

Comment:
Since we're going with "an N-API", perhaps we should not leave out the indefinite article here either.